tests/main_app/utils/wikitext/temps_bot/test_temps_bot.py

```
@@ -19,3 +24,178 @@ def test_get_titles_from_wikilinks(sample_from_prompt: str):
+        text = """
+        {{SVGLanguages|test-file,World,2020.svg}}
+        {{owidslidersrcs|File:title1.svg!year=2020}}
+        """
+        main_title, titles = get_files_list(text)
+        assert main_title == "test-file,World,2020.svg"
+        assert "title1.svg" in titles
+
+    def test_main_title_underscores_to_spaces(self) -> None:
+        """Test that underscores in main title are converted to spaces."""
+        text = "{{SVGLanguages|test_file_name.svg}}"
+        main_title, titles = get_files_list(text)
```

Ruff (0.15.5)

[warning] Unpacked variable `titles` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)